en/complete_vehicles/px4_vision_kit.md

```diff
@@ -0,0 +1,299 @@
+- The *globalplanner* has not been tested with this kit.
+- The *landing planner* requires a downward facing camera, and cannot used without first modifying the camera mounting.
+
+PX4 and the companion computer exchange data over [MAVLink](https://mavlink.io/en/) using these interfaces:
+- [Path Planning Interface](../computer_vision/path_planning_interface.md) - API for implementing avoidance features in automatic modes.
+- [Collision Prevention Interface](../computer_vision/collision_prevention.md) - API for vehicle based avoidance in manual position mode based on an obstacle map (currently used for collision prevention).
+
+### Boot the Companion Computer {#boot_mission_computer}
+
+The first three steps of [Fly the Drone (with avoidance)](#fly_drone) explain how to boot the companion computer.
+
+In summary, you just insert the provided USB stick into the *UP Core* port labeled `USB1`, and then power the vehicle using a 4S battery.
+The USB stick contains a bootable *Ubuntu Linux 18.04* image with PX4 Avoidance software included, which automatically runs the *local planner* for object avoidance and collision prevention on startup.
+
+The avoidance system will be up and running after about 2.5 minutes.
+
+Once started the companion computer can be used both as a computer vision development environment and for running the software.
+
+
+### Login to the Companion Computer {#login_mission_computer}
+
+To login to the companion computer:
+1. Connect a keyboard and mouse to the *UP Core* via port `USB2`:
+
+   ![UP Core: USB2](../../assets/hardware/px4_vision_devkit/upcore_port_usb2.png)
+   - Use the USB-JST cable from the kit to get a USB A connector
+   
+     ![USB to JST cable](../../assets/hardware/px4_vision_devkit/usb_jst_cable.jpg)
+   - A USB hub can be attached to the cable if the keyboard and mouse have separate connectors. 
+1. Connect a monitor to the *UP Core* HDMI port.
+
+   ![UP Core: HDMI port](../../assets/hardware/px4_vision_devkit/upcore_port_hdmi.png)
+
+   The Ubuntu login screen should then appear on the monitor. 
+1. Login to the *UP Core* using the credentials:
+   - **Username:** px4vision
+   - **Password:** px4vision
+
+### Installing the image on the Companion Computer {#install_image_mission_computer}
+
+You can alternatively install the image on the *UP Core* and boot from internal memory (instead of the USB).
+
+To flash the USB image to the *UP Core*:
+
+1. Insert the pre-flashed USB drive into the *UP Core* port labeled `USB1`.
+1. [Login to the companion computer](#login_mission_computer) (as described above).
+1. Open a terminal and run the following command to copy the image onto internal memory (eMMC).
+   The terminal will prompt for a number of responses during the flashing process.
+   ```sh
+   cd ~/catkin_ws/src/px4vision_ros
+   sudo ./flash_emmc.sh
+   ```
+
+   > **Note** All information saved in the *UP Core* computer will be removed when executing this script
```

Comment:
Maybe adding that the boot order might change so that the vehicle will no longer boot from the usb-stick might be helpful?